.github/workflows/release.yml

```diff
@@ -49,11 +49,11 @@ jobs:
                   echo "FEAT_NAME=" >> $GITHUB_ENV
                   echo "TAG_NAME=prerelease" >> $GITHUB_ENV
             - run: npm ci
-            - name: vsix
+            - name: vsix # TODO: For packages/toolkit release only
```

Comment:
Is the comment describing the current situation or a future situation? This prerelease job can and should start producing multiple artifacts as soon as we can do so--no need to wait on that.